### PR TITLE
NEW: ADD circulation_pump

### LINKED ISSUE #8: Add CIRCULATION_PUMP component status
Tells of the filtration cycle is running. And together with HEATING ON:OFF, it an be used to know if it filtration cycle or just because heating is require.
    {
      componentType: 'CIRCULATION_PUMP',
      materialType: 'CIRCULATION_PUMP',
      value: 'OFF',
      availableValues: [Array],
      name: 'CIRCULATION_PUMP',
      registeredTimestamp: '2023-01-21T06:14:49.181+0000'
    },

### PR DESCRIPTION
**Pull request by martijnpoppen**
FIX: #8

### CODE REVIEW

File: drivers/main-device.js [hunk 1]
```diff
@@ -229,7 +236,8 @@ module.exports = class mainDevice extends Homey.Device {
             const blower0 = await this.getComponent('BLOWER', components, '0');
             const blower1 = await this.getComponent('BLOWER', components, '1');
             const blower2 = await this.getComponent('BLOWER', components, '2');
-
+            const circulationPump = await this.getComponent('CIRCULATION_PUMP', components);
+console.log(circulationPump)
```
Comment (oh2th) [inline drivers/main-device.js]: Assuming this is only for development and in final would go away.

**Review by oh2th — APPROVED** [review]
Looks good

File: drivers/main-device.js (hunk 1 above)
Comment (martijnpoppen) [reply drivers/main-device.js]: hehe yes

**Comment by oh2th** [comment]
Found an issue missing capability measure_circulation_pump from driver.compose.json?

Never mind, the branch I'm testing against is broken. It has partially the measure_circulation_pump available. so all good.

**Pull request merged**